src/asm2wasm.h

```diff
@@ -1331,7 +1331,8 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
 
   // apply memory growth, if relevant
   if (preprocessor.memoryGrowth) {
-    emscripten::generateMemoryGrowthFunction(wasm);
+    EmscriptenGlueLinker emscripten(wasm, Address(0));
```

Comment:
maybe `emscripten => glueLinker`?

Reply:
To be honest I'm not entirely enthused about the class name "EmscriptenGlueLinker", but I can't think of what else to call it. EmscriptenGluer? Emscriptenifier?

But also secretly these were so I could change `emscripten::` into `emscripten.`. Barring a better class name, `glueLinker` is better here yeah.